p2p/discover/table.go

```diff
@@ -44,6 +44,10 @@ const (
 
 	maxBondingPingPongs = 16
 	maxFindnodeFailures = 5
+
+	autoRefreshInterval = 1 * time.Hour
+	seedCount           = 30
+	seedMaxAge          = 5 * 24 * time.Hour
```

Comment:
I added this because I noticed that most of the seed nodes found in my database had a last pong time of more than a week ago. This can happen because the first expiration happens 1h after startup. Nodes that haven't been contacted for that long are unlikely to be still alive.